### PR TITLE
SISRP-18352 - 5.4 My Finances (Student) - Financial Resources

### PR DESCRIPTION
**Pull request by davidlimcheng**
https://jira.berkeley.edu/browse/SISRP-18352

Will update release-specific instructions upon merge.

### CODE REVIEW

File: public/json/campuslinks.json [hunk 1]
```diff
@@ -1751,8 +1751,26 @@
       ]
     },
     {
+      "name": "Dream Act Application",
```
Comment (raydavis) [inline public/json/campuslinks.json]: Sorry, @davidlimcheng , I bet this convention isn't documented, but the campuslinks.json is in strict alphabetical order by "name". Mind moving this entry?

Comment (davidlimcheng) [inline public/json/campuslinks.json]: @raydavis Ah, sure thing.

**Comment by davidlimcheng** [comment]
Amended w/ alphabetization & a little capitalization clean-up for consistency.

**Comment by raydavis** [comment]
Thanks, @davidlimcheng

**Pull request merged**